charts/k8s-monitoring/docs/ScrapeApplicationMetrics.md

```diff
@@ -0,0 +1,205 @@
+
+Here is an example that filters to a service named "database", in the namespace "blue", with the port named "metrics":
+
+```river
+discovery.relabel "blue_database_service" {
+  targets = discovery.kubernetes.services.targets  // Gets all services
+  rule {  // Keep all services named "database"...
+    source_labels = ["__meta_kubernetes_service_name"]
+    regex = "database"
+    action = "keep"
+  }
+  rule {  // ... that exist in the "blue" namespace...
+    source_labels = ["__meta_kubernetes_namespace"]
+    regex = "blue"
+    action = "keep"
+  }
+  rule {  // ... and only scrape its port named "metrics".
+    source_labels = ["__meta_kubernetes_service_port_name"]
+    regex = "metrics"
+    action = "keep"
+  }
+}
+```
+
+The [documentation](https://grafana.com/docs/agent/latest/flow/reference/components/discovery.kubernetes/#service-role)
+has the list of meta labels for services. 
+
+This is also a good place to add any extra labels that will be scraped. For example, if you wanted to set the label
+`team="blue"`, you might use this additional rule:
+
+```river
+  rule {
+    target_label = "team"
+    action = "replace"
+    replacement = "blue"
+  }
+```
+
+### Pod discovery
+
+Similar to service discovery, we use a [`discovery.relabel`](https://grafana.com/docs/agent/latest/flow/reference/components/discovery.relabel/)
+component to select the specific pod or pods that we want to scrape. The [meta labels for pods](https://grafana.com/docs/agent/latest/flow/reference/components/discovery.kubernetes/#pod-role)
+will be slightly different, but the concept is the same.
+
+Here is an example that filters to a specific set of pods that starts with name "analysis", with the label "system.component=image":
+
+```river
+discovery.relabel "image_analysis_pods" {
+  targets = discovery.kubernetes.pods.targets  // Gets all pods
+  rule {  // Keep all pods named "analysis.*"...
+    source_labels = ["__meta_kubernetes_pod_name"]
+    regex = "analysis.*"
+    action = "keep"
+  }
+  rule {  // ... with the label system.component=image
+    source_labels = ["__meta_kubernetes_pod_label_system_component"]
+    regex = "image"
+    action = "keep"
+  }
+}
+```
+
+Note that there is a unique meta label for every Kubernetes label. The labels are prefixed with
+`__meta_kubernetes_pod_label_` and the label name is normalized so all non-alphanumeric characters become underscores (`_`).
+
+## Scraping
+
+Now that we've selected the specific pod or service we want, we can scrape it for metrics. This is done with the
+[`prometheus.scrape`](https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.scrape/) component. At its basic, you only need to declare what things to scrape, and where to send
+the scraped metrics. Here is an example:
+
+```river
+prometheus.scrape "processing_app" {
+  targets = discovery.relabel.image_analysis_pods.targets
+  forward_to = [prometheus.remote_write.grafana_cloud_prometheus.receiver]
```

Comment:
Maybe a little comment to say something like: see "Delivery" for more on remote_write